client/services/tables/tables.filter.directive.js

```diff
@@ -0,0 +1,14 @@
+angular.module('probedock.tables').directive('filters', function (stConfig, $timeout) {
```

Comment:
I would name this directive `stFilters` or `tableFilters` so that its role is clearer when used in the template.